pdf_build_src/process_markdowns.py

```diff
@@ -447,14 +447,11 @@ def process_macros(duplicated_src_dir_path):
             matches = re.findall(re.compile("({{.*?}})", re.DOTALL), contents)
             for m in matches:
                 # Remove macro delimiters to get *just* the function call
-                function_string = m.strip("{} ")
-                # Replace prefix with module name
-                function_string = function_string.replace(
-                    "MACROS___",
-                    "macros."
-                )
-                # Run the function to get the output
-                new = eval(function_string)
+                function_name = m.strip("{} ")
+                # Remove prefix
+                function_name = function_name.replace("MACROS___", "")
+                # Find function_name in imported module macros
+                new = getattr(macros, function_name)

```

Comment:
Didn't analyze but CI seems to report

```
Traceback (most recent call last):
  File "/root/project/pdf_build_src/process_markdowns.py", line 470, in <module>
    process_macros(duplicated_src_dir_path)
  File "/root/project/pdf_build_src/process_markdowns.py", line 454, in process_macros
    new = getattr(macros, function_name)
AttributeError: module 'schemacode.macros' has no attribute 'make_metadata_table(
   {
        "LongName": "OPTIONAL",
        "Description": (
            "RECOMMENDED",
            "The description of the column.",
        ),
        "Levels": "RECOMMENDED",
        "Units": "RECOMMENDED",
        "TermURL": "RECOMMENDED",
        "HED": "OPTIONAL",
   }
)'

Exited with code exit status 1
```

Reply:
Was the error there in my first attempt? Will look into it again.

Reply:
I have totally misunderstood what `eval`() is doing here. Will close this PR.